OpenKeychain/src/main/java/org/sufficientlysecure/keychain/keyimport/ParcelableEncryptedKeyRing.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2016 Alex Fong Jie Wen <alexfongg@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.sufficientlysecure.keychain.keyimport;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.util.Pair;
+
+import java.util.ArrayList;
+
+/**
+ * Parcelable representation of an encrypted keyring block as raw data,
+ * with related fields not obtainable without decrypting the block
+ */
+public class ParcelableEncryptedKeyRing implements Parcelable{
```

Comment:
Woops I should fix that. Sounds pretty confusing now that I read that comment ><